app/views/content/life-as-a-teacher/explore-subjects/modern-languages/_article.html.erb

```diff
@@ -0,0 +1,149 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach languages</h2>
+    <p>As a language teacher, you’ll teach the linguistic aspects of language and introduce pupils to the cultural contexts that give language its richness and depth.</p>
+    <p>There is a huge range of languages you can teach, from Japanese and Mandarin to French and Spanish, and you may find that you teach ore than one language.</p> 
+      <p>In addition to the personal rewards of teaching languages, there are practical benefits too, including:</p>
+      <ul>
+        <li>a <a href="/life-as-a-teacher/pay-and-benefits">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
```

Comment:
```suggestion
        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
```

Normally go to the pay page directly rather than the sub-category?